SO_properties.py

```diff
@@ -758,7 +758,10 @@ def Xrayphlum_no_agn(self):
     def compY_no_agn(self):
         if self.Ngas == 0:
             return None
-        return self.gas_compY[self.gas_no_agn].sum().value * self.compY_unit
```

Comment:
I don't quite understand how this is a problem. If all elements of self.gas_no_agn are False it should just return zero.

Reply:
This is to handle self.gas_no_agn=None. In the xray branch we've ensured that never happens. Probably best to leave this fix in here.